appeals/web/src/server/appeals/appeal-details/manage-linked-appeals/__tests__/manage-linked-appeals.test.js

```diff
@@ -71,17 +71,17 @@ describe('linked-appeals', () => {
 	});
 	afterEach(teardown);
 
-	describe('GET /change-appeal-type/appeal-type', () => {
-		it('should render the decision page lead', async () => {
+	describe('GET /linked-appeals/manage', () => {
+		it('should render the manage linked appeals page with the expected content when the appeal is a lead', async () => {
```

Comment:
Can we check the expected content so it is clear what that content should be?

Reply:
good idea, it makes sense to start doing that

Reply:
Done!